examples/cookbook/agents/web_search_agent/messages.py

```diff
@@ -113,3 +113,27 @@
         "name": "extract_content",
     },
 ]
+
+test_extract_content_messages = [
+    {"role": "user", "content": "What is the capital of France?"},
+    {
+        "role": "assistant",
+        "content": "",
+        "tool_calls": [
+            {
+                "type": "function",
+                "function": {
+                    "arguments": '{"queries":["capital of France","capital city of France","France","Paris","France capital"]}',
+                    "name": "_web_search",
+                },
+                "id": "call_ddhSpVcNeDg9bLz4C7HEw85P",
+            }
+        ],
+    },
+    {
+        "role": "tool",
+        "content": "h\nt\nt\np\ns\n:\n/\n/\ne\nn\n.\nw\ni\nk\ni\np\ne\nd\ni\na\n.\no\nr\ng\n/\nw\ni\nk\ni\n/\nP\na\nr\ni\ns\n\nh\nt\nt\np\ns\n:\n/\n/\nw\nw\nw\n.\nb\nr\ni\nt\na\nn\nn\ni\nc\na\n.\nc\no\nm\n/\np\nl\na\nc\ne\n/\nP\na\nr\ni\ns\n\nh\nt\nt\np\ns\n:\n/\n/\ne\nn\n.\nw\ni\nk\ni\np\ne\nd\ni\na\n.\no\nr\ng\n/\nw\ni\nk\ni\n/\nP\na\nr\ni\ns\n\nh\nt\nt\np\ns\n:\n/\n/\nw\nw\nw\n.\nb\nr\ni\nt\na\nn\nn\ni\nc\na\n.\nc\no\nm\n/\np\nl\na\nc\ne\n/\nP\na\nr\ni\ns\n\nh\nt\nt\np\ns\n:\n/\n/\ne\nn\n.\nw\ni\nk\ni\np\ne\nd\ni\na\n.\no\nr\ng\n/\nw\ni\nk\ni\n/\nF\nr\na\nn\nc\ne\n\nh\nt\nt\np\ns\n:\n/\n/\nw\nw\nw\n.\nb\nr\ni\nt\na\nn\nn\ni\nc\na\n.\nc\no\nm\n/\np\nl\na\nc\ne\n/\nF\nr\na\nn\nc\ne\n\nh\nt\nt\np\ns\n:\n/\n/\ne\nn\n.\nw\ni\nk\ni\np\ne\nd\ni\na\n.\no\nr\ng\n/\nw\ni\nk\ni\n/\nP\na\nr\ni\ns\n\nh\nt\nt\np\ns\n:\n/\n/\nw\nw\nw\n.\nb\nr\ni\nt\na\nn\nn\ni\nc\na\n.\nc\no\nm\n/\np\nl\na\nc\ne\n/\nP\na\nr\ni\ns\n\nh\nt\nt\np\ns\n:\n/\n/\nw\nw\nw\n.\nb\nr\ni\nt\na\nn\nn\ni\nc\na\n.\nc\no\nm\n/\np\nl\na\nc\ne\n/\nP\na\nr\ni\ns\n\nh\nt\nt\np\ns\n:\n/\n/\ne\nn\n.\nw\ni\nk\ni\np\ne\nd\ni\na\n.\no\nr\ng\n/\nw\ni\nk\ni\n/\nP\na\nr\ni\ns",
```

Comment:
woah what's going on here with all the newlines?

Reply:
i think thats how the url gets formatted? i can look deeper if needed. but the extract_content properly gets the url from this so this might be benign?

Reply:
wait, i found the issue will fix

Reply:
done